.github/workflows/pytest-gpu.yml

```diff
@@ -46,31 +46,34 @@ jobs:
           pytest-gpu-acc-nvidia,
           pytest-gpu-omp-amd
         ]
+        test_examples: ["examples/seismic/tti/tti_example.py examples/seismic/acoustic/acoustic_example.py examples/seismic/viscoacoustic/viscoacoustic_example.py examples/seismic/viscoelastic/viscoelastic_example.py examples/seismic/elastic/elastic_example.py"]
+
         include:
         - name: pytest-gpu-omp-nvidia
           test_files: "tests/test_adjoint.py tests/test_gpu_common.py tests/test_gpu_openmp.py"
```

Comment:
Now that they are the same for all builds, we can pull this variable outside the `include` scope, and maybe put it right under the newly introduced `test_examples` ?

Reply:
They are not the openacc build is different

Reply:
ah right !